jenkins/model_output_processing_Jenkinsfile

```diff
@@ -28,8 +28,12 @@ pipeline {
         // May need to actually use yesterday's date depending on what time of day this runs
         //sh 'DATE=$(date -d "yesterday 13:00" "+%Y_%m_%d"); wget https://owi-common-resources.s3-us-west-2.amazonaws.com/resources/application/nhm/output/data/climate_${DATE}.nc'
         sh '''
+          PATTERN="*.nc"
+          FILES=($PATTERN)
+          DATE_PART="${FILES:0:10}"
+          DATE_PRETTY="${DATE_PART//_/-}"
           wget https://prod-owi-resources.s3-us-west-2.amazonaws.com/resources/Application/wbeep/model_output/all_quantiles.rds
-          DATE=2019_08_22
+          DATE=DATE_PART
           wget https://owi-common-resources.s3-us-west-2.amazonaws.com/resources/application/nhm/output/data/climate_${DATE}.nc
```

Comment:
@wdwatkins can i delete this line then? we don't need the climate data?

Reply:
yes